File: Selenium/Activities/Activity15.java
package activities;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
        import org.openqa.selenium.WebDriver;
        import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
        import org.openqa.selenium.firefox.FirefoxOptions;

public class Activity15 {
    public static void main(String[] args) {
        // Set up the Chrome WebDriver
        WebDriverManager.chromedriver().setup();
        ChromeDriver driver= new ChromeDriver();
        // Open the webpage
        driver.get("https://v1.training-support.net/selenium/dynamic-attributes");

        // Get the title of the page and print it
        String pageTitle = driver.getTitle();
        System.out.println("Page Title: " + pageTitle);

        // Find the username and password input fields
        WebElement usernameField = driver.findElement(By.xpath("//input[@type='text']"));
        WebElement passwordField = driver.findElement(By.xpath("//input[@type='password']"));

        // Type in the credentials
        usernameField.sendKeys("admin");
        passwordField.sendKeys("password");

        // Find and click the submit button
        driver.findElement(By.xpath("//button[@type='submit']")).click();

        // Print the login message
        String message = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Login message: " + message);

        // Close the browser
        driver.quit();
    }
}

File: Selenium/Activities/Activity15.py
from  selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.chrome.service import Service as ChromeService
from webdriver_manager.chrome import ChromeDriverManager
driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))

# Open the webpage
driver.get("https://v1.training-support.net/selenium/dynamic-attributes")

# Get the title of the page and print it
pageTitle = driver.title
print("Page Title:", pageTitle)

# Find the username and password input fields
usernameField = driver.find_element(By.XPATH, "//input[@type='text']")
passwordField = driver.find_element(By.XPATH, "//input[@type='password']")

# Type in the credentials
usernameField.send_keys("admin")
passwordField.send_keys("password")

# Find the login button and click it
driver.find_element(By.XPATH, "//button[@type='submit']").click()

# wait for the login message to appear
#wait.until(EC.visibility_of_element_located((By.ID, "action-confirmation")))

# Print the confirmation message
message = driver.find_element(By.ID, "action-confirmation").text
print("Login message: ", message)
